pika/heartbeat.py

```diff
@@ -12,21 +12,24 @@ class HeartbeatChecker(object):
     intervals.
 
     """
+    DEFAULT_INTERVAL = 600
```

Comment:
It's been 60 seconds by default in the broker and most clients for a good couple of years now.